package.json

```diff
@@ -30,25 +32,31 @@
   "license": "MIT",
   "scripts": {
     "build": "cheminfo-build --entry src/index.js --root SmartArrayFilter",
```

Comment:
This won't work (see https://github.com/mljs/global-spectral-deconvolution/blob/e9e8522996bbe435e7778dd910c304cdfc578cae/package.json#L17)

Reply:
fixed can you check this line too please